crates/polars-ops/src/series/ops/fused.rs

```diff
@@ -41,17 +41,20 @@ fn fma_ca<T: PolarsNumericType>(
     ChunkedArray::from_chunk_iter(a.name().clone(), chunks)
 }
 
-pub fn fma_series(a: &Series, b: &Series, c: &Series) -> Series {
+pub fn fma_columns(a: &Column, b: &Column, c: &Column) -> Column {
     if a.len() == b.len() && a.len() == c.len() {
         with_match_physical_numeric_polars_type!(a.dtype(), |$T| {
-            let a: &ChunkedArray<$T> = a.as_ref().as_ref().as_ref();
-            let b: &ChunkedArray<$T> = b.as_ref().as_ref().as_ref();
-            let c: &ChunkedArray<$T> = c.as_ref().as_ref().as_ref();
+            let a: &ChunkedArray<$T> = a.as_materialized_series().as_ref().as_ref().as_ref();
```

Comment:
If we can materialize the scalar to a `series` of length 1, we will have very large speed ups here as the arithmetic can deal with broadcasting already.